app/javascript/components/lease_ups/LeaseUpApplicationsPage.js

```diff
@@ -16,8 +17,8 @@ const LeaseUpApplicationsPage = ({ listing, applications }) => {
       link: `/listings/${listing.id}/lease-ups/export`
     },
     breadcrumbs:[
-      {title: 'Lease Ups',  link: '/lease-ups'},
-      {title: listing.name, link: `/listings/${listing.id}/lease-ups`}
+      {title: 'Lease Ups',  link: '/listings/lease-ups'},
+      {title: listing.name, link: appPaths.toListingLeaseUps(listing.id)}
```

Comment:
@fddayan what's the difference between toLeaseUpApplications and toListingLeaseups? they're duplicates right now

Reply:
Interesting, I think it should be `toLeaseUpApplications` everywhere

Reply:
ok I'll update on this one